lib/experimental/fonts/font-library/font-library.php

```diff
@@ -32,9 +32,45 @@ function gutenberg_init_font_library_routes() {
 	);
 	register_post_type( 'wp_font_family', $args );
 
+	register_post_type(
+		'wp_font_face',
+		array(
+			'labels'       => array(
+				'name'          => __( 'Font Faces', 'gutenberg' ),
+				'singular_name' => __( 'Font Face', 'gutenberg' ),
+			),
+			'public'       => false,
+			'_builtin'     => true,                              /* internal use only. don't use this when registering your own post type. */
+			'hierarchical' => false,
+			'show_in_rest' => false,
```

Comment:
Not sure why `show_in_rest` is set to `false`. It seems contradictory with the function defined after

```
	protected function check_is_post_type_allowed( $post_type ) {
	     return true;
	}
```

Reply:
I'm thinking of it this way: we don't want to manage the wp_font_face post type through the REST API, in the way that one usually would for post types that store user created content. From the perspective of the API consumer, posts for font faces and font families is more of an implementation detail. For example, none of the fields used in other post related endpoints (`post_title`, `post_content`, etc) are used in the REST API interface for fonts. We don't want autosaves or revisions, which are typically used with `'show_in_rest' => true,`.

This also matches how the `wp_global_styles` post type is registered, which I think is the one most similar to `wp_font_face` and `wp_font_family`. These 3 post types store settings, rather than content.

Another problem is that a side effect of `'show_in_rest' => true,` is that the post type is added to the `wp/v2/types` endpoint, including links that will be incorrect for font faces (`wp/v2/font-faces` rather than `wp/v2/font-families/<id>/font-faces`). I don't see a way to correct that with filters, currently.